packages/web/src/scss/components/Tabs/index.html

```diff
@@ -0,0 +1,45 @@
+{{#> layout }}
+  <h2 class="typography-heading-medium-text mt-1100 mb-800">Tabs</h2>
+  <ul class="Tabs" role="tablist">
+    <li class="Tabs__item">
+      <a
+        href="#pane1"
+        class="Tabs__link is-selected"
+        role="tab"
+        aria-selected="true"
+        id="pane1-tab"
+        data-toggle="tab"
+        data-target="#pane1"
+        aria-controls="pane1"
+      >
+        Item Selected
+      </a>
+    </li>
+    <li class="Tabs__item">
+      <a
+        href="#pane2"
+        class="Tabs__link"
+        role="tab"
+        aria-selected="false"
+        id="pane2-tab"
+        data-toggle="tab"
+        data-target="#pane2"
+        aria-controls="pane2"
+      >
+        Item
+      </a>
+    </li>
+    <li class="Tabs__item">
+      <a href="https://www.example.com" class="Tabs__link">Item Link</a>
+    </li>
+    <li class="Tabs__item Tabs__item--desktop--visible">
+      <a href="https://www.example.com" class="Tabs__link">Item Link Only Desktop</a>
+    </li>
+  </ul>
+  <div class="Tabs-content">
```

Comment:
Looks like a dummy class 😐. I think we can get rid of the `div` here and in the README since this is just a basic example with no further assumptions.